packages/icons/src/IconNames.ts

```diff
@@ -1346,8 +1346,8 @@ export const enum IconNames {
   HalfCircle = 'HalfCircle',
   QuarterCircle = 'QuarterCircle',
   ThreeQuarterCircle = 'ThreeQuarterCircle',
-  SixPointStar = '6PointStar',
-  TwelvePointStar = '12PointStar',
+  SixPointStar = 'SixPointStar',
+  TwelvePointStar = 'TwelvePointStar',

```

Comment:
This is a fair change, but just for context, the 6/12 names come from the icons team. That said, the icon tool can be updated to use these aliases and be more consistent. I'll do that today.

Reply:
should we add aliases for these?